tools/codegen/gofilegen/schema/data_source_schema_file.go

```diff
@@ -0,0 +1,41 @@
+package schema
+
+import (
+	"fmt"
+	"go/format"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/tools/codegen/codespec"
+	"github.com/mongodb/terraform-provider-mongodbatlas/tools/codegen/gofilegen/codetemplate"
+)
+
+// GenerateDataSourceSchemaGoCode generates the data_source_schema.go file containing
+// DataSourceSchema() and TFDSModel for data sources.
+func GenerateDataSourceSchemaGoCode(input *codespec.Resource) ([]byte, error) {
```

Comment:
similar to `schema_file_test.go` and `resource_file_test.go` should we have a unit test for data source files?